### DISCUSSION TITLE
How do I allow the nether roof?

### DISCUSSION THREAD

**Discussion by Razphiz** [GriefPrevention/GriefPrevention#1813] [Q&A] [answered]
In my rules of my server I stated that going to the roof of the nether is allowed, but they can't go there because of GriefPrevention. I checked config.yml and I didn't see any settings for it. Is this something I can allow somehow?

**Comment by Jikoo** [comment, marked as answer]
GP does not block access to the nether roof. You may wish to check either your anticheat's settings or your Purpur config.

**Reply by Razphiz** [reply]
It was a different plugin called IllegalStack, thanks.
